.gitattributes

```diff
@@ -0,0 +1,5 @@
+*       text=auto
+*.cs    text diff=csharp
+*.md    text
+*.yml   text
+*.props text eol=crlf
```

Comment:
Should we also do .targets as well?

Reply:
I don't see any .targets files yet, but we could preemptively add it if there are plans to add one in the future.